### PR TITLE
1 to 1 reqs

### PR DESCRIPTION
**Pull request by KyraWayne**
Styling for 1 to 1 reqs

### CODE REVIEW

**Comment by KyraWayne** [comment]
@jkingsman

File: js/classloader.js [hunk 1]
```diff
@@ -71,21 +71,32 @@ function fillColumn(colNumber) {
...
+            reqEntryDiv.className = 'reqEntryDiv pull-right';
 
+            //checkbox creation
+            //dropdown and checkbox styling
             var requirementCheckbox = document.createElement('input');
             requirementCheckbox.setAttribute('type', 'checkbox');
             requirementCheckbox.id = requirementName;
             requirementCheckbox.className = classes + ' pull-right requirementMarker';
-            requirementCheckbox.style.display = 'none';
+            requirementCheckbox.style.marginRight = '5px';
```
Comment (jkingsman) [inline js/classloader.js]: Can we add a class and put it in the stylesheet? In-JS styles are really hard to manage/track down where styling is coming from.

File: js/classloader.js [hunk 2]
```diff
@@ -71,21 +71,32 @@ function fillColumn(colNumber) {
...
-            requirementEntry.appendChild(requirementCheckbox);
-            requirementEntry.appendChild(requirementDropdown);
-            requirementEntry.appendChild(classSatisfactionDropdown);
+            requirementEntry.appendChild(reqEntryDiv);
+            reqEntryDiv.appendChild(requirementDropdown);
+            reqEntryDiv.appendChild(requirementCheckbox);
+
+            if(classSatisfactionDropdown.length > 2){
+                reqEntryDiv.appendChild(classSatisfactionDropdown);
+                requirementCheckbox.style.display = 'none';
```
Comment (jkingsman) [inline js/classloader.js]: Same as above - can we put this into a class?

File: js/helpers.js [hunk 3]
```diff
@@ -222,7 +222,7 @@ function restoreStatus() {
         requirementDropdown.value = elective.date;
 
         requirementEntry.appendChild(requirementLabel);
-        requirementEntry.appendChild(document.createElement('br'));
+       // requirementEntry.appendChild(document.createElement('br'));
```
Comment (jkingsman) [inline js/helpers.js]: If we don't need this, just go ahead and delete it -- no need to comment code unless it's going to be used later.

**Review by jkingsman — CHANGES_REQUESTED** [review]
Looks great conceptually - just a couple places to move styling out of the JS and into CSS and a comment deletion.

**Comment by KyraWayne** [comment]
@jkingsman see new commits

File: js/classloader.js [hunk 4]
```diff
@@ -71,21 +71,30 @@ function fillColumn(colNumber) {
...
-            requirementEntry.appendChild(requirementCheckbox);
-            requirementEntry.appendChild(requirementDropdown);
-            requirementEntry.appendChild(classSatisfactionDropdown);
+            requirementEntry.appendChild(reqEntryDiv);
+            reqEntryDiv.appendChild(requirementDropdown);
+            reqEntryDiv.appendChild(requirementCheckbox);
+
+            if(classSatisfactionDropdown.length > 2){
+                reqEntryDiv.classList.add('hasClassDropdown');
+                reqEntryDiv.appendChild(classSatisfactionDropdown);
```
Comment (jkingsman) [inline js/classloader.js]: Can we append no matter what and just have it set to `display: none`? It breaks some other stuff that has to do with knowing when to and when not to display the satisfying class by the requirement in the calendar (note all the undefineds now).

**Review by jkingsman — CHANGES_REQUESTED** [review]

**Comment by jkingsman** [comment]
Looks great; awesome.

**Pull request merged**